server/datastore/mysql/migrations/tables/20240617174138_UpdateMobileConfigColumnType.go

```diff
@@ -0,0 +1,26 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20240617174138, Down_20240617174138)
+}
+
+func Up_20240617174138(tx *sql.Tx) error {
+	stmt := `
+ALTER TABLE mdm_apple_configuration_profiles MODIFY COLUMN mobileconfig MEDIUMBLOB;
```

Comment:
Just double-checking as I don't know offhand, did you verify that it leaves the `NOT NULL` requirement as-is?

Reply:
Also, in case others wonder about it too, the Windows `syncml` column is already a `MEDIUMBLOB` (and the MDM declarations are `JSON`, which is apparently equivalent to `LONGBLOB/LONGTEXT`). So no such concerns for size on those tables.

Reply:
Looking at `schema.sql`, looks like it did remove the `NOT NULL` constraint, you should add it to the migration so that it is maintained.

Reply:
Ah great catch! fixed.